rust/arrow/src/buffer.rs

```diff
@@ -919,24 +962,18 @@ mod tests {
 
     #[test]
     fn test_from_raw_parts() {
-        let buf = unsafe { Buffer::from_raw_parts(null_mut(), 0, 0) };
```

Comment:
why was this test removed? Because it is no longer possible to create a buffer from a null pointer?

Reply:
Exactly :)